### ISSUE TITLE
List differences between uber/react-map-gl vs urbica/react-map-gl

### ISSUE THREAD

**Issue by NameFILIP** [urbica/react-map-gl#240]
Currently, the docs say this package was inspired by uber/react-map-gl. I would like to know what are the conceptual differences between `uber/react-map-gl` vs `urbica/react-map-gl`. Would it be possible to add such info somewhere in the docs to help others choose between these two alternatives.

**Comment by NameFILIP** [comment]
👋 any updates on this? I've noticed the event types are closer to mapbox-gl's. Would you be able to list the key differences here to start with, before putting the into the docs?

**Comment by derwaldgeist** [comment]
I want to second on this. I just found this package and would love to migrate, since you also provide a cluster option that the original does not offer. I'd love to see a list of differences, or maybe a migration guide?

**Comment by gilneto8** [comment]
Any update? I'm also looking into the clustering capabilities and I'm feeling tempted to migrate.

**Comment by 7200rpm** [comment]
Agree. It says inspired by the uber, but not clear how this one is meant to be different. A quick explanation on the readme would be really helpful!

**Comment by derwaldgeist** [comment]
I did the switch and never looked back. Can only recommend to do it.

**Comment by gilneto8** [comment]
The only downside I got aware of is the lack of Typescript support.
